jdaviz/core/helpers.py

```diff
@@ -137,13 +137,20 @@ def plugins(self):
 
         # handle renamed plugins during deprecation
         if 'Orientation' in plugins:
+            # renamed in 3.9
             plugins['Links Control'] = plugins['Orientation']._obj.user_api
             plugins['Links Control']._deprecation_msg = 'in the future, the formerly named \"Links Control\" plugin will only be available by its new name: \"Orientation\".'  # noqa
         if 'Canvas Rotation' in plugins:
+            # removed in 3.9
             plugins['Canvas Rotation']._deprecation_msg = 'this functionality will be removed in favor of the implementation for rotation in the \"Orientation\" plugin.'  # noqa
         if 'Export' in plugins:
+            # renamed in 3.9
             plugins['Export Plot'] = plugins['Export']._obj.user_api
             plugins['Export Plot']._deprecation_msg = 'in the future, the formerly named \"Export Plot\" plugin will only be available by its new name: \"Export\".'  # noqa
```

Comment:
I think these have now been more than 2 releases, so maybe we can remove for 4.0?

Reply:
(I don't think that belongs in scope here, so I made a note in the 4.0 release ticket)